CorsixTH/Lua/ui.lua

```diff
@@ -803,7 +804,7 @@ function UI:afterLoad(old, new)
 
   if old < 100 then
     self:removeKeyHandler({"alt", "enter"}, self)
-    self:addKeyHandler({"alt", "Return"}, self, self.toggleFullscreen)
+    self:addKeyHandler({"alt", "return"}, self, self.toggleFullscreen)
```

Comment:
It is just 'consistency', cause every other addKeyHandler is using lowercase key-names, i know addKeyHandler will lowercase all key-names, too. So this is not necessary, but i did it. I hope it does not make problems with the afterLoad stuff for old savegames